### DISCUSSION TITLE
Improve thresholds (possibly improve values and more tests covered)

### DISCUSSION THREAD

**Discussion by vojtechhuser** [OHDSI/DataQualityDashboard#212] [General]
DQD team conducted a network study.
It was submitted to AMIA conference.

The results are here:
https://github.com/vojtechhuser/DataQuality/blob/master/extras/DqdResults/S01-benchmark-kb-subset.csv
and
https://github.com/vojtechhuser/DataQuality/blob/master/extras/DqdResults/

Clair and Frank need to provide guidance how I should format the KB so that it is best for digestion by DQD.    E.g., DQD computes p1 and p99 and the DQD code compares the ones that are off.

**Comment by fdefalco** [comment]
improvements in process in measurement-palooza branch.
